models/git/protected_branch.go

```diff
@@ -314,8 +314,8 @@ type WhitelistOptions struct {
 // This function also performs check if whitelist user and team's IDs have been changed
 // to avoid unnecessary whitelist delete and regenerate.
 func UpdateProtectBranch(ctx context.Context, repo *repo_model.Repository, protectBranch *ProtectedBranch, opts WhitelistOptions) (err error) {
-	if err = repo.GetOwner(ctx); err != nil {
-		return fmt.Errorf("GetOwner: %v", err)
+	if err = repo.LoadOwner(ctx); err != nil {
+		return fmt.Errorf("LoadOwner: %v", err)
```

Comment:
How did that make it in?
I'll accept it for this PR as it seems to be a problem throughout this whole file…
It should be
```go
		return fmt.Errorf("LoadOwner: %w", err)
```

Reply:
![image](https://user-images.githubusercontent.com/18380374/219856014-a19c1151-f486-4657-b566-a4a28fcda6a1.png)
it seems that this problem is not only in this file.

Reply:
I know.
(Although many of these will be `log.X("%v", err)`, where it won't apply.)
That's what I meant with "I accept it for this PR".